test/integration-tests/shared/parseEvents.ts

```diff
@@ -20,3 +20,4 @@ export function parseEvents(iface: Interface, receipt: TransactionReceipt): (Log
     })
     .filter((n: LogDescription | undefined) => n)
 }
+
```

Comment:
Remove please

Reply:
remove file parseEvents?

Reply:
No, no. I am only about empty line, but I already mentioned in some comment, that it's better to use some fmt script